libs/features/mas/web-components/test/merch-datasource.test.html.js

```diff
@@ -44,6 +44,7 @@ runTests(async () => {
             const [ccCard, photoshopCard] = getTemplateContent('cards');
             document.querySelector('main').append(ccCard, photoshopCard);
             expect(aemMock.count).to.equal(0);
+            const card = document.querySelector('main merch-card:has(> merch-datasource[path="/content/dam/sandbox/mas/creative-cloud"])');
```

Comment:
we should stop using `/content/dam/sandbox/mas` and start using `/content/dam/mas`

Reply:
will address in next PR